app/components/specific/map/RaceMap.js

```diff
@@ -3,24 +3,16 @@ import PropTypes from "prop-types";
 import { Dimensions } from "react-native";
 import MapView from "react-native-maps";
 
+import positionHelper from "../../../store/services/helpers/gps";
 import styles from "./styles";
 import theme from "../../../theme/theme";
 
 const { width, height } = Dimensions.get("window");
 
 const ASPECT_RATIO = width / height;
-const LATITUDE = 42.78386;
-const LONGITUDE = 0.15844;
-const LATITUDE_DELTA = 0.37;
+const LATITUDE_DELTA = 0.57;
```

Comment:
what is latitude delta?

Reply:
value used to compute initial region (maps).
this has to be computed depending on event map. (to be done).